src/expression.rs

```diff
@@ -127,6 +131,7 @@ lazy_static! {
         PrecClimber::new(vec![
             Operator::new(add, Left) | Operator::new(subtract, Left),
             Operator::new(multiply, Left) | Operator::new(divide, Left) | Operator::new(rem, Left),
+            Operator::new(lshift, Left) | Operator::new(rshift, Left),
```

Comment:
I think C has a lower precedence for right shift and left shift than addition and subtraction. Since the `PrecClimber` array is specified from lower to higher precedence (in line with the name *climb*), the shift operators should be first in the list.

Reply:
Ah, of course! I just force pushed the correct precedence and updated tests

Reply:
I know it doesn't really matter the order they are mentioned in the grammar or in match arms, but should I update them there too?

Reply:
Nope, it's good to go. :smile: